dpdata/vasp/outcar.py

```diff
@@ -136,3 +160,45 @@ def analyze_block(lines, ntot, nelm) :
                 coord.append(info[:3])
                 force.append(info[3:6])
     return coord, cell, energy, force, virial, is_converge
+
+def analyze_block_ml(lines, ntot) :
+    coord = []
+    cell = []
+    energy = None
+    force = []
+    virial = None
+    for idx,ii in enumerate(lines) :
```

Comment:
This line does not obey [PEP-8](https://peps.python.org/pep-0008/).

Reply:
deleted space beforce the colon